src/packagedcode/buildpack.py

```diff
@@ -0,0 +1,152 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import toml
+from packagedcode import models
+from packageurl import PackageURL
+import yaml
+
+class BuildpackHandler(models.NonAssemblableDatafileHandler):
+    """
+    Handle buildpack.toml manifests.
+    See https://buildpacks.io/ for details on buildpack format.
+    """
+    datasource_id = "buildpack_toml"
+    path_patterns = ("*buildpack.toml",)
+    default_package_type = "generic"
+    description = "Cloud Native Buildpack manifest"
+    documentation_url = "https://buildpacks.io/"
+
+    @classmethod
+    def parse(cls, location, package_only=False):
+        """
+        Parse the buildpack.toml file at `location` and yield PackageData.
+        """
+        with open(location, "r", encoding="utf-8") as f:
+            data = toml.load(f)
+
+        api_version = data.get("api")
+        buildpack = data.get("buildpack", {})
+        if not buildpack:
+            return
+
+        buildpack_id = buildpack.get("id")
+        name = buildpack.get("name")
+
+        if buildpack_id:
+            namespace, name = buildpack_id.split("/", 1)
+        
+        # Initialize common package data
+        package_data = dict(
+            datasource_id=cls.datasource_id,
+            type=cls.default_package_type,
+            name=name,
+            version=buildpack.get("version"),
+            description=buildpack.get("description"),
+            homepage_url=buildpack.get("homepage"),
+            keywords=buildpack.get("keywords", []),
+            extracted_license_statement=None,
+            dependencies=[],
```

Comment:
You don't need to add `dependencies` and `extra_data` like this, as these are initialized by default. Remove these two lines.